Source/Scene/BillboardCollection.js

```diff
@@ -1176,6 +1197,10 @@ define([
         }
 
         var disableDepthTestDistance = billboard.disableDepthTestDistance;
+        if (billboard.heightReference === HeightReference.CLAMP_TO_GROUND && disableDepthTestDistance === 0.0) {
+            disableDepthTestDistance = 2000.0;
```

Comment:
Should this be configurable at the scene level?  I think there are other billboard/label/depth-ish options there.

Reply:
It's configurable by setting `disableDepthTestDistance` to a different value